modules/web/test/com/haulmont/charts/model/amcharts/charts/container/ChartsTestContainer.java

```diff
@@ -0,0 +1,35 @@
+package com.haulmont.charts.model.amcharts.charts.container;
+
+import com.haulmont.cuba.web.testsupport.TestContainer;
+
+public class ChartsTestContainer extends TestContainer {
```

Comment:
Are you sure that we need all this staff?

Reply:
Yes, because we need to get converters from Datatypes util method which uses DatatypeRegistry bean which reads datatypes from datatypes.xml config. Moreover, locale is taken from the user session (or metadata bean), so I have to have all these beans in the context. Mocking is not a great solution here because it doesn't use formatters specified in the datatypes.xml.